### PR TITLE
Update chapter4.txt

### PR DESCRIPTION
**Pull request by forkingOut**
Hope you like it.

### CODE REVIEW

**Comment by forkingOut** [comment]
Hope you like it

File: chapter4.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ This is a plot where we get oscor.
 =======
 This is a plot where we get oscor.
 >>>>>>> love-triangle-plot
```
Comment (sahilgupta098) [inline chapter4.txt:5]: yeah its right

**Review by sahilgupta098 — APPROVED** [review]

**Comment by sahilgupta098** [comment]
Approved!

File: chapter4.txt (hunk 1 above)
Comment (forkingOut) [reply chapter4.txt:5]: glad you like it.

**Pull request merged**